minigalaxy/ui/window.py

```diff
@@ -67,7 +67,11 @@ def __init__(self, name="Minigalaxy"):
             os.makedirs(THUMBNAIL_DIR, mode=0o755)
 
         # Interact with the API
-        self.__authenticate()
+        try:
+            self.__authenticate()
+        except Exception as e:
+            print(e)
+            self.offline = True
         self.HeaderBar.set_subtitle(self.api.get_user_info())
```

Comment:
Wouldn't this cause an exception in this case?

Reply:
No, I don't think. From what I saw, the Login dialog stay open as long as the authentication is not successful.